### PR TITLE
docs(templates): adds Ticket Templates

### PR DESCRIPTION
**Pull request by pon-james**
Adds templates for three ticket types (Bug, Story, Task, Epic).

### CODE REVIEW

File: templates/jira/Bug.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Bug ticket template
+
+Use this template for reporting a bug or regression. It is important to that the input describes what is observed, what is expected and how to reproduce the problem. Provide as much information as possible for a speedy resolution.
```
Comment (zeger-pon) [inline templates/jira/Bug.md:3]: Suggest: keep it factual (or something similar to keep emotions out of it)

File: templates/jira/Epic.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Epic ticket template
+
+An Epic is a big feature or task. It can span multiple sprints and can be broken down into smaller tasks. Use Epic tickets to group related tasks together into logical collections.
```
Comment (zeger-pon) [inline templates/jira/Epic.md:3]: Can we quantify "big"? Or give some examples?

File: templates/jira/Epic.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+Here it is important to introduce the "job to be done" at a high level and explain the why. The Epic should be broken down into smaller tasks that can be completed in a single sprint.
```
Comment (zeger-pon) [inline templates/jira/Epic.md:5]: An epic can span multiple sprints?

File: templates/jira/Task.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
+# Task ticket template
+
+Use this template for describing a task. It is important to that the input describes the scope or boundaries of the change (e.g. which part of the solution), the acceptance criteria and the stakeholders that need to be informed. Provide as much input as possible for a good briefing.
```
Comment (zeger-pon) [inline templates/jira/Task.md:3]: So a task is a 'regular' ticket?

File: templates/jira/Bug.md (hunk 1 above)
Comment (pon-james) [reply templates/jira/Bug.md:3]: Added "observed objectively". See: https://github.com/pondevelopment/pon-developer-guide/pull/11/commits/ef14329caf5859cef42c096e41348600cc568590

File: templates/jira/Epic.md (hunk 2 above)
Comment (pon-james) [reply templates/jira/Epic.md:3]: Added "(spans multiple sprints)" see: https://github.com/pondevelopment/pon-developer-guide/commit/ef14329caf5859cef42c096e41348600cc568590

**Review by zeger-pon — APPROVED** [review]

**Review by roy-pon — APPROVED** [review]

**Pull request merged**